020-Getting-started/080-astro.mdx

```diff
@@ -1,46 +1,47 @@
 ---
-title: Astro with Xata starter guide
+title: Get started with Astro and Xata
 navTitle: Astro
-keywords: ['astro']
+keywords: ['astro', 'full-text search']
 description: Get started with Astro and Xata
 slug: getting-started/astro
 published: true
 ---
 
-In this tutorial we'll create a new Xata database and query it using [Astro][2]. Before you begin you will need a user account. If you haven't done so already, please [sign up or log in][0].
+In this guide, we'll create a new [Astro][2] application and walk through adding Xata database and search
```

Comment:
This suggestion would update the text of this sentence to be:

> In this guide, we'll create a new [Astro][2] application and walk through adding a Xata database and search capabilities functionality.

Maybe the fact this is over multiple lines was missed? Unless clarified, I'll stick with the existing text if that's okay.

> In this guide, we'll create a new [Astro][2] application and walk through adding Xata database and search functionality.

Reply:
> In this guide, we'll create a new [Astro][2] application and walk through adding Xata database and search functionality.

This isn't singular because it's "database and search" -> "adding Xata database and search functionality". So, I think this is correct.

"adding a Xata database and search functionality" would be incorrect.

Please let me know if I'm missing something.